membership_extension/models/res_partner.py

```diff
@@ -21,6 +21,16 @@ class ResPartner(models.Model):
     _inherit = "res.partner"
 
     associate_member = fields.Many2one(index=True)
+    is_adhered_member = fields.Boolean(
+        string="Adhered member",
+        help="A member who is associated to another one, but whose membership "
+             "are independent.",
```

Comment:
Maybe with this new feature, you must change help of associate_member, because now the date is independent. I'm right?

Reply:
Not really. What an adhered member sees is the field `membership_start_adhered` wich will written onto it's `membership_start` when computed. And that field has its own help (*Date from which partner is adhered*)